_posts/2017-04-13-bell-hooks-feminism-is-for-everybody.md

```diff
@@ -0,0 +1,177 @@
+
+### 8: Global Feminism
+Feminism exists in a "white supremacist capitalist patriarchal Western culture" and is not immune from its power dynamics. At its inception, the feminist movement was not about women gaining freedom from patriarchal oppression globally. Instead, it was used by women of privilege to gain more class mobility - who then wanted to "reach out" to women of other cultures to "save" them from their oppression. All of this without a political commitment to end Western imperialist policies and transnational capitalism that are economically and militaristically oppressing these women.
+
+A radical feminist perspective instead examines how sexist practices globally are linked non-hierarchically. Instead of saying, "your sexism is worse than ours, so we can save you," we can recognize "your symptoms of patriarchy are different from ours, but they are ALL bad and come from the same root - so we can all struggle together."  
+
+### 9: Women at Work
+The emphasis of women gaining access to the workplace was a _huge_ point of contention in the early feminist movement. Imagine a bunch of white, middle class feminists chanting, "work will liberate you!" to a group of poor and working-class women who have _always_ worked for low wages and did not in any way feel "liberated" by that work.
+
+While it is true that _economic self-sufficiency_ is needed for liberation, _work_ by itself does not liberate women from sexist oppression and does not even provide economic self-sufficiency for many poor working-class people. Feminist politics is also a politics of working-class people, male and female: addressing ways to reduce poverty and improve the material conditions for the poorest people in our society, many of whom happen to be female.
+
+### 10: Race and Gender
+For a long time, the feminist movement has ignored issues of race: The fact that in America, _whiteness_ is a manufactured and privileged category, and that any movement that ignores the unique impact of racial oppression of non-white women cannot claim to speak for women as a group. Feminist movements cannot continue to deny this difference and place the needs of white women over the needs of others. 
+
+### 11: Ending Violence
+Domestic violence was one of the most positive interventions made by the contemporary feminist movement, and bell hooks draws attention to its connection with sexist oppression by calling it "patriarchal violence."
+
+>The term "patriarchal violence" is useful because unlike the more accepted phrase "domestic
```

Comment:
Try removing [_base.scss#L179](https://github.com/radicalbarracuda/sf-feminist-book-club/blob/master/_sass/_base.scss#L179) to get the line-height to adjust properly. Didn't realize I left this line.